### PR TITLE
Add measurement_service.proto v2

### PR DESCRIPTION
**Pull request by WesleyTangNationalInstruments**
- [X] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/measurement-services-python/blob/main/CONTRIBUTING.md).

### What does this Pull Request accomplish?

We [added annotations to Output parameter to measurement_services.proto V2](https://ni.visualstudio.com/DevCentral/_git/ASW/pullrequest/474704). This PR propagates that changes to the `measurementlink-python` repo.

We copy the proto file over and ran `generate_grpc_stubs.py`.

Note that this PR is just to ensure that the `measurementlink-python` repo has the latest measurement_service.proto and is in sync with the "source of truth" for the proto file. The python measurement service does not yet use the v2 proto file, but it looks like there's some [active work](https://github.com/ni/measurementlink-python/compare/main...users/bkeryan/measurement-v2) on that front. The InstrumentStudio squad has a [a future work item](https://ni.visualstudio.com/DevCentral/_backlogs/backlog/InstrumentStudio%20Platform/Features/?workitem=2364515) to update the python measurement service to support enums via annotations using the features added to this v2 proto file.

### Why should this Pull Request be merged?

[AB#2371125](https://ni.visualstudio.com/DevCentral/_backlogs/backlog/InstrumentStudio%20Platform/Features/?workitem=2371125)

### What testing has been done?

N/A.

### CODE REVIEW

File: ni_measurementlink_service/_internal/stubs/session_pb2.py [hunk 1]
```diff
@@ -19,7 +19,6 @@
 _builder.BuildMessageAndEnumDescriptors(DESCRIPTOR, globals())
 _builder.BuildTopDescriptorsAndMessages(DESCRIPTOR, "session_pb2", globals())
 if _descriptor._USE_C_DESCRIPTORS == False:
-
     DESCRIPTOR._options = None
```
Comment (bkeryan) [inline ni_measurementlink_service/_internal/stubs/session_pb2.py:22]: FYI, I'm going to update generate_grpc_stubs.py to stop reformatting with Black. We already exclude the stubs in our Black and ni-python-styleguide configurations.

File: ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/measurement/v2/measurement_service.proto [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  // See documentation for ConfigurationParameter annotations for more details and examples.
+  // Note that enum annotations only apply to TYPE_ENUM parameters and not TYPE_STRING parameters.
+  // Well-known annotations:
+  // - Type specialization. The keys to other annotations will be read based on the value of `ni/type_specialization` annotation.
+  //   - Key: "ni/type_specialization"
+  //   - Common Values: "enum" ...    
+  // - For string parameter with ni/type_specialization annotation equals "enum"
+  //   - Key: "ni/enum.values"
+  //   - Expected format: JSON dictionary string"
+  //   - Example: "{\"RED\":0, \"GREEN\":25, \"BLUE\":5}"
```
Comment (bkeryan) [inline ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/measurement/v2/measurement_service.proto:158]: FYI, I posted my comments on this on the corresponding Azure Devops PR (albeit late)

**Review by bkeryan — APPROVED** [review]

**Comment by bkeryan** [comment]
Spelling: "propagate"

But really, "Add measurement_services.proto v2" is clearer.

**Review by dixonjoel — APPROVED** [review]

File: ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/measurement/v2/measurement_service.proto (hunk 2 above)
Comment (WesleyTangNationalInstruments) [reply ni_measurementlink_service/_internal/stubs/proto/ni/measurementlink/measurement/v2/measurement_service.proto]: [Addressed those comments in ASW](https://ni.visualstudio.com/DevCentral/_git/ASW/pullrequest/475543) and pulled the changes into this PR.

File: ni_measurementlink_service/_internal/stubs/session_pb2.py (hunk 1 above)
Comment (bkeryan) [reply ni_measurementlink_service/_internal/stubs/session_pb2.py:22]: https://github.com/ni/measurementlink-python/pull/247

**Pull request merged**